src/sap.ui.core/src/sap/ui/base/ManagedObject.js

```diff
@@ -4823,6 +4823,8 @@ sap.ui.define([
 	 * <li>The boolean value <code>cloneChildren</code> specifies whether associations/aggregations will be cloned</li>
 	 * <li>The boolean value <code>cloneBindings</code> specifies if bindings will be cloned</li>
 	 * </ul>
+	 * When <code>oOptions</code> is undefined both options default to <code>true</code>.
+	 * But be aware that setting the optject and not specifying eigher <code>cloneChildren</code> or <code>cloneBindings</code> will result in them beeing undefined and therefor set to <code>false</code>
```

Comment:
"optject": typo or interesting new word creation? It's at least not common to use that term in OpenUI5 and I only find few references to it in the web.

Reply:
that happens when you start thinking in the middle of the word and then choose another (options/object)

Reply:
Hey @sebbi08 let us provide a change for that :)